http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala

```diff
@@ -110,6 +119,25 @@ class NonLeoMessageSubscriber[F[_]: Timer](gkeAlg: GKEAlgebra[F],
           }
         } yield ()
     } yield ()
+
+  private[monitor] def handleDeleteNodepoolMessage(
```

Comment:
Should this also update the nodepool status to `DELETED` in the DB? (Similar to `handleDeleteKubernetesClusterMessage`)

Reply:
ah yes!

Reply:
or actually this method calls `gkeInterp.deleteAndPollNodepool` which marks the nodepool as deleted in our DB so we should be good. going to test that now